install-nimbus-cli.sh

```diff
@@ -0,0 +1,284 @@
+#!/usr/bin/env bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+: "${BINARY_NAME:=nimbus-cli}"
+: "${TASKCLUSTER_HOST:=https://firefox-ci-tc.services.mozilla.com}"
+: "${USE_SUDO:=false}"
+: "${SHELL:=bash}"
+: "${DEBUG:=false}"
+: "${NIMBUS_INSTALL_DIR:=""}"
+
+# This is largely inspired by the MIT-licensed get-helm-3 install script
+# https://github.com/helm/helm/blob/main/scripts/get-helm-3
```

Comment:
Great choice, also I 10/10 recommend Helm if you do any k8s work 😅 

My comments will be based on a diff between the helm script and this one, I'm trusting that Helm's install script works well (from experience).